etc/scripts/clean-jupyterlab.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+#
+# Copyright 2018-2020 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+ANACONDA_HOME=${ANACONDA_HOME:='~/opt/anaconda'}
```

Comment:
The default should match the default path anaconda uses which is directly in user home not in /opt
```suggestion
ANACONDA_HOME=${ANACONDA_HOME:='~/anaconda'}
```